-fオプションによる画像ファイルの指定でエラーが発生

https://github.com/fgophi/fgosccnt/pull/115 にアップデート後、-fオプションがエラーにより使用できなくなりました。
以前は同じコマンドで、同じディレクトリを指定できていました。
```
$ python fgosccnt.py -f O:\Pictures\FGO\result\2020.7_ハンティング\01_ワイバーンハント_典位+級
```
```
Traceback (most recent call last):
  File "fgosccnt.py", line 1748, in <module>
    csvfieldnames, outputcsv = get_output(inputs, args.debug)
  File "fgosccnt.py", line 1692, in get_output
    if not output['filename'].endswith(': Not Found') and \
AttributeError: 'WindowsPath' object has no attribute 'endswith'
```

ここ数日アップデートしていなかったため、どのバージョンから使用できなくなったかは不明です。

エラーメッセージが変わりました

```
Traceback (most recent call last):
  File "fgosccnt.py", line 1648, in get_output
    sc = ScreenShot(img_rgb, svm, svm_chest, svm_card, fileextention, debug)
  File "fgosccnt.py", line 422, in __init__
    self.items.append(Item(item_img_rgb, item_img_gray, svm, svm_card, fileextention, mode, debug))
  File "fgosccnt.py", line 816, in __init__
    self.ocr_digit(mode, debug)
  File "fgosccnt.py", line 1345, in ocr_digit
    self.dropnum =  self.detect_white_char(base_line, margin_right, font_size, debug=debug) + self.dropnum
  File "fgosccnt.py", line 1169, in detect_white_char
    line = self.get_number(base_line, margin_right, font_size)
  File "fgosccnt.py", line 1103, in get_number
    result = self.read_char(pt)
  File "fgosccnt.py", line 1287, in read_char
    tmpimg = cv2.resize(tmpimg, (win_size))
cv2.error: OpenCV(4.2.0) C:\projects\opencv-python\opencv\modules\imgproc\src\resize.cpp:4045: error: (-215:Assertion failed) !ssize.empty() in function 'cv::re
size'


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "fgosccnt.py", line 1748, in <module>
    csvfieldnames, outputcsv = get_output(inputs, args.debug)
  File "fgosccnt.py", line 1681, in get_output
    output = ({'filename': filename + ': not valid'})
TypeError: unsupported operand type(s) for +: 'WindowsPath' and 'str'
```

--- a/fgosccnt.py
+++ b/fgosccnt.py
@@ -1639,7 +1639,7 @@ def get_output(filenames, debug=False):
         f = Path(filename)
 
         if f.exists() == False:
-            output = { 'filename': filename + ': Not Found' }
+            output = { 'filename': str(filename) + ': Not Found' }
         else:
             img_rgb = imread(filename)
             fileextention = Path(filename).suffix
@@ -1678,7 +1678,7 @@ def get_output(filenames, debug=False):
                     output["ドロ数"] = str(output["ドロ数"]) + "+"
                 output.update(sc.allitemdic)
             except:
-                output = ({'filename': filename + ': not valid'})
+                output = ({'filename': str(filename) + ': not valid'})
         outputcsv.append(output)
     new_outputcsv = []
     if ce_drop == True:

